### DISCUSSION TITLE
Resource Missing

### DISCUSSION THREAD

**Discussion by alie-samuel** [PokeAPI/pokeapi#732] [Q&A] [answered]
Hi! I'm developing PokeAPI Wrapper for Delphi and the resource is missing:

https://pokeapi.co/api/v2/super-contest-effect/3

Can i just ignore this one in my unit tests?

**Comment by Naramsim** [comment, marked as answer]
Yeah, ofc! Sorry for the late reply. Submit the wrapper to us when you're done with it.

**Reply by alie-samuel** [reply]
Thank You! The Wrapper is complete. Link to the repository and some examples: https://github.com/devsa707/PokeWrapper4D
